openraft/src/defensive.rs

```diff
@@ -343,3 +300,59 @@ where
         Ok(())
     }
 }
+
+pub fn check_range_matches_entries<D: AppData, RNG: RangeBounds<u64> + Debug + Send>(
+    range: RNG,
```

Comment:
RNG is usually short for random number generator

Reply:
I agree, but there is a type that may conflict with it: `std::ops::Range`...

Reply:
How about just `RB`

Reply:
Nice~